.travis.yml

```diff
@@ -90,6 +90,8 @@ matrix:
           - hdf5
           - eigen
           - ninja
+    - os: windows
+      env: USE_XTENSOR=False
```

Comment:
Xtensor seems to install fine in windows. Is there any problem compiling with support?

Reply:
I invoke @tristan0x one more time.